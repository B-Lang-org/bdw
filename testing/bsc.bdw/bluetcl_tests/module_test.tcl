namespace import ::Bluetcl::*

# Sets the specified flag
puts [flags set {-verilog}]

# Loads the specified module
puts [module load mkT]

# Clears the curently loaded module
puts [module clear]
