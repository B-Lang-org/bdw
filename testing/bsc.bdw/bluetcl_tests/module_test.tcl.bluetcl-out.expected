Welcome to Blueshell
Version: 2008.01; Date: 2008-01-31; Revision 12993
-verilog
mkM mkT

